core/miner/storage_fsm/impl/precommit_batcher_impl.hpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <boost/multiprecision/cpp_int.hpp>
+#include <chrono>
+#include <libp2p/protocol/common/asio/asio_scheduler.hpp>
+#include <libp2p/protocol/common/scheduler.hpp>
+#include <map>
+#include <vm/actor/actor.hpp>
+#include "api/full_node/node_api.hpp"
+#include "miner/storage_fsm/precommit_batcher.hpp"
+#include "miner/storage_fsm/types.hpp"
+#include "primitives/address/address.hpp"
+#include "primitives/types.hpp"
+#include "vm/actor/actor_method.hpp"
+#include "vm/actor/builtin/types/miner/sector_info.hpp"
+
+namespace fc::mining {
+  using api::FullNodeApi;
+  using api::SectorNumber;
+  using api::SectorPreCommitInfo;
```

Comment:
Already used in recommit_batcher.hpp, remove

Reply:
fixed